src/Objects/Printer.vala

```diff
@@ -167,7 +167,8 @@ public class Printers.Printer : GLib.Object {
      */
     public string info {
         get {
-            return CUPS.get_option ("printer-info", dest.options) ?? "";
+            unowned string? info = CUPS.get_option ("printer-info", dest.options);
+            return info != null && info != "" ? info : "Unnamed printer";
```

Comment:
You need to make the string translatable and have "printer" title case.
```suggestion
            return info != null && info != "" ? info : _("Unnamed Printer");
```

Reply:
Changes applied